moveit_ros/moveit_servo/include/moveit_servo/collision_monitor.hpp

```diff
@@ -1,12 +1,7 @@
 /*******************************************************************************
- *      Title     : servo.h
- *      Project   : moveit_servo
- *      Created   : 3/9/2017
- *      Author    : Brian O'Neil, Andy Zelenak, Blake Anderson
- *
  * BSD 3-Clause License
  *
- * Copyright (c) 2019, Los Alamos National Security, LLC
+ * Copyright (c) 2023, PickNik Inc.
```

Comment:
Changing license holders is not cool.

```suggestion
 * Copyright (c) 2019, Los Alamos National Security, LLC
```

Reply:
oh, sorry about that !!